pkg/agent/core/ngt/config/config_test.go

```diff
@@ -98,6 +98,7 @@ func TestNewConfig(t *testing.T) {
 			if err := test.checkFunc(test.want, gotCfg, err); err != nil {
 				tt.Errorf("error = %v", err)
 			}
+
```

Comment:
it doesn't affect the codes but... why these blank lines appeared?

Reply:
I don't know